src/applications/vre/28-8832/config/form.js

```diff
@@ -18,13 +18,25 @@ import {
 } from './chapters/claimant-information';
 import { isDependent } from './helpers';
 
+import manifest from '../manifest.json';
+
 const formConfig = {
+  rootUrl: manifest.rootUrl,
   urlPrefix: '/',
   submitUrl: '/v0/education_career_counseling_claims',
   trackingPrefix: '28-8832-planning-and-career-guidance-',
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
   formId: VA_FORM_IDS.FORM_28_8832,
+  saveInProgress: {
+    messages: {
+      inProgress:
+        'Your Personalized Career Planning and Guidance Chapter 36 benefit is in progress.',
+      expired:
+        'Your saved Personalized Career Planning and Guidance Chapter 36 benefit has expired. If you want to apply for PCPG Chapter 36 benefits, please start a new application.',
```

Comment:
Same here, update to `Your Personalized Career Planning and Guidance Chapter 36 benefits application`